src/utils/elapsedTime.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any,@typescript-eslint/no-unsafe-assignment,@typescript-eslint/no-unsafe-return */
+
+import { Logger, LoggerLevel } from '@salesforce/core';
+import { LoggerLevelValue } from '@salesforce/core/lib/logger/logger';
+
+const logThis = (
```

Comment:
nit: this always makes for confusion in a js/ts world.  Maybe just call this `log`?   I could see this getting exported for use as need across the module. (doesn't have to be now, just when needed.)